Content.Server/Communications/CommunicationsConsoleSystem.cs

```diff
@@ -52,6 +52,9 @@ public override void Initialize()
             SubscribeLocalEvent<CommunicationsConsoleComponent, CommunicationsConsoleAnnounceMessage>(OnAnnounceMessage);
             SubscribeLocalEvent<CommunicationsConsoleComponent, CommunicationsConsoleCallEmergencyShuttleMessage>(OnCallShuttleMessage);
             SubscribeLocalEvent<CommunicationsConsoleComponent, CommunicationsConsoleRecallEmergencyShuttleMessage>(OnRecallShuttleMessage);
+
+            // On console init, set cooldown
+            SubscribeLocalEvent<CommunicationsConsoleComponent, ComponentStartup>((uid, comp, _) => OnCommunicationsConsoleInit(comp));
         }
```

Comment:
This breaks with serialization as it gets bulldozed every time, use timeoffsetserializer.

Reply:
Didn't know this existed, thank you! I'm refactoring now. I think I can remove the side effect of not being able to announce immediately on map init.

Reply:
After looking into this, I think the Comms Console needs to be refactored for the TimeOffSetSerializer and TimeSpan to work properly with it. I wasn't able to get it working in its current state. I will keep looking into it.